src/Pim/Bundle/CatalogBundle/tests/integration/ElasticSearch/IndexConfiguration/PimCatalogNumberIntegration.php

```diff
@@ -0,0 +1,302 @@
+<?php
+
+namespace Pim\Bundle\CatalogBundle\tests\integration\Elasticsearch\IndexConfiguration;
+
+/**
+ * This integration tests checks that given an index configuration and some products indexed
+ * the text area research is consistent.
+ *
+ * @author    Samir Boulil <samir.boulil@akeneo.com>
+ * @author    Anaël Chardan <anael.chardan@akeneo.com>
+ * @author    Philippe Mossière <philippe.mossiere@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class PimCatalogNumberIntegration extends AbstractPimCatalogIntegration
+{
+    public function testLowerThanOperator()
+    {
+        $query = [
+            'query' => [
+                'bool' => [
+                    'filter' => [
+                        'range' => [
+                            'values.box_quantity-decimal.<all_locales>.<all_channels>' => ['lt' => 10],
```

Comment:
shouldn't we give strings?
maybe we should test it works with both strings and numbers (just need to dupplicate the tests)
same remark everywhere